operator/src/trustee.rs

```diff
@@ -176,10 +187,12 @@ pub async fn recompute_reference_values(ctx: RvContextData) -> anyhow::Result<()
     let image_pcrs_map = operator_config_maps.get(PCR_CONFIG_MAP).await?;
     let image_pcrs = get_image_pcrs(image_pcrs_map)?;
     // TODO many grub+shim:many OS image recompute once supported
-    let mut reference_values_in = BTreeMap::from([(
-        "svn".to_string(),
-        vec![serde_json::Value::String("1".to_string())],
-    )]);
+    let mut reference_values_in = BTreeMap::from([
+        (
```

Comment:
There are a lot of changes in the PR that are formatting only (e.g. this one), but the code was formatted right, which is why the format CI is failing. Please use `cargo fmt` with a recent version (1.85+) and default config.